### PR TITLE
Clean up the calculus section a bit

### PR DESCRIPTION
**Pull request by asmeurer**
- Move the integrals to the top
- Focus on the algorithms for each function
- Remove the discussion of unevaluated functions, and just put a single
  sentence about it at the end (with an example)
- Use a slightly more complicated example for derivatives
- Describe the "algorithm" for derivatives
- Consistency in the paragraphs (first sentences names the function,
  subsequent sentences describe the algorithms, then example)

### CODE REVIEW

**Comment by asmeurer** [comment]
Still TODO: we need better integration examples (#138), and summation and product examples.

File: calculus.tex [hunk 1]
```diff
@@ -1,63 +1,40 @@
...
-Unevaluated expressions can be evaluated with the \verb|doit| method.
-\begin{verbatim}
->>> expr.doit()
-cos(x)
-\end{verbatim}
-
-% TODO: A more interesting example here
-Integrals can be analogously, calculated either with the \verb|integrate|
-function, or the unevaluated \verb|Integral| objects.
+Integrals are calculated either with the \verb|integrate| function. SymPy
```
Comment (certik) [inline calculus.tex]: ...or with the `\verb{Integral}` function?

File: calculus.tex [hunk 2]
```diff
@@ -71,3 +48,11 @@
 >>> limit((2*E**((1-cos(x))/sin(x))-1)**(sinh(x)/atan(x)**2), x, 0)
 E
 \end{verbatim}
```
Comment (certik) [inline calculus.tex:50]: Reference the section in the supplement here for more details about Limits.

File: calculus.tex (hunk 1 above)
Comment (asmeurer) [inline calculus.tex]: I've removed all discussion of unevaluated objects from the descriptions here. There is just the paragraph about it at the bottom of this section.

**Comment by certik** [comment]
Otherwise +1

File: calculus.tex (hunk 1 above)
Comment (asmeurer) [inline calculus.tex]: Oh I see. I forgot to remove "either"

Comment (certik) [inline calculus.tex]: Yes, remove "either".

File: calculus.tex (hunk 2 above)
Comment (certik) [inline calculus.tex:50]: Maybe we shouldn't reference it here, but rather in the introduction.

Comment (certik) [inline calculus.tex:50]: I made #141 for this.

**Pull request merged**